book/run/observability.md

```diff
@@ -43,15 +47,72 @@ brew services start prometheus
 brew services start grafana
 ```
 
+#### Linux (Debian/Ubuntu)
+
+For Prometheus:
+
+```bash
+# Download the latest version of Prometheus
+wget https://github.com/prometheus/prometheus/releases/download/v2.45.0/prometheus-2.45.0.linux-amd64.tar.gz
+
+# Extract the archive
+tar xvfz prometheus-*.tar.gz
+cd prometheus-*
+
+# Start Prometheus (in the background)
+./prometheus --config.file=prometheus.yml > prometheus.log 2>&1 &
+```
+
+For Grafana:
```

Comment:
maybe we reference the official install instructions instead, then we dont need to maintain this